tests/Feature/Sensors/JobAttemptSensorTest.php

```diff
@@ -712,7 +712,7 @@ public function test_it_does_not_ingest_or_build_up_state_while_idle($workComman
 
         $this->assertGreaterThan(50, $loops);
         $ingest->assertWrittenTimes(0);
-        $this->assertCount(2, $this->core->ingest->buffer);  // popping query + illuminate:queue:restart
+        $this->assertCount(0, $this->core->ingest->buffer);
```

Comment:
Does this mean that we have lost data that we wouldn't previously?

Surprised this change has occurred but not sure if I've missed something.